biorxiv/journal_tracker/map_preprints_to_journals.py

```diff
@@ -240,39 +206,45 @@ def call_pmc(doi_ids):
 print(len(missing_ids))
 
 
-# In[12]:
+# In[10]:
 
 
 chunksize=100
 data = []
 for chunk in tqdm.tqdm(range(0, len(missing_ids), chunksize)):
     query_ids = missing_ids[chunk:chunk+chunksize]
-    response = call_pmc(query_ids).json()
+    response = call_pmc(query_ids, 'annorxivr', 'dnicholson329@gmail.com').json()
```

Comment:
> Do you want to pull your email from a config file, or is it okay to have it here since it's in a notebook?

Ack thanks for catching. Yeah I need to pull that out. Don't want random users frauding me lol.